torch/autograd/graph.py

```diff
@@ -258,8 +259,7 @@ def __exit__(self, *args: object):
 
 
 class save_on_cpu(saved_tensors_hooks):
-    """Context-manager under which tensors saved by the forward pass will be
-    stored on cpu, then retrieved for backward.
+    """Context-manager that saves tensors on CPU during forward pass then retrieval in backward.
```

Comment:
I think it is ok to keep the phrasing here, it doesn't seem like pydocstyle flagged this other than the missing period

```suggestion
    """Context manager  under which tensors saved by the forward pass will be stored on cpu, then retrieved for backward.
```

Reply:
Hi @mikaylagawarecki the purpose of I rephrase  the docstring is to set the length of docstring is within the column length of 100, what I mean is:
![image](https://github.com/pytorch/pytorch/assets/23013350/a7cb71a7-f0ef-479a-b783-6ebaac6230b2)

 I thought this is pytorch project's setting from `.vscode` but turned out it was my own setting. I would revert it :)